src/cascadia/QueryExtension/ExtensionPalette.idl

```diff
@@ -36,7 +36,16 @@ namespace Microsoft.Terminal.Query.Extension
         Windows.UI.Xaml.Controls.IconElement ResolvedIcon { get; };
 
         event Windows.Foundation.TypedEventHandler<ExtensionPalette, IInspectable> ActiveControlInfoRequested;
-        event Windows.Foundation.TypedEventHandler<ExtensionPalette, IInspectable> AIKeyAndEndpointRequested;
         event Windows.Foundation.TypedEventHandler<ExtensionPalette, String> InputSuggestionRequested;
     }
+
+    [default_interface] runtimeclass TerminalContext : IContext
+    {
+        TerminalContext(String activeCommandline);
+    }
+
+    [default_interface] runtimeclass SystemResponse : IResponse
```

Comment:
I guess, similarly - will we treat different `IResponse`'s differently? Seems like they'd all be a {message, error}, regardless of the source... right?

Reply:
Same as above